nautobot/core/settings.py

```diff
@@ -296,7 +296,10 @@
         "HOST": os.getenv("NAUTOBOT_DB_HOST", "localhost"),
         "PORT": os.getenv("NAUTOBOT_DB_PORT", ""),
         "CONN_MAX_AGE": int(os.getenv("NAUTOBOT_DB_TIMEOUT", "300")),
-        "ENGINE": os.getenv("NAUTOBOT_DB_ENGINE", "django.db.backends.postgresql"),
+        "ENGINE": os.getenv(
+            "NAUTOBOT_DB_ENGINE",
+            "django_prometheus.db.backends.postgresql" if METRICS_ENABLED else "django.db.backends.postgresql",
```

Comment:
Can you add a note in the `required-settings.md` `## DATABASES` section about this change/requirement?